shared/tracker/bio.render.desktop.js

```diff
@@ -58,7 +59,7 @@ const styles = {
     backgroundColor: colors.greyBackground,
     justifyContent: 'flex-start',
     paddingTop: 12,
-    width: 202
+    width: featureFlags.get('tracker') === 'v2'? 320 : 202
```

Comment:
if its v2 it doesn't need an explicit width. if we can make our sub components as responsive as possible that's better so in v2 it just takes the full width